package.json

```diff
@@ -7,14 +7,15 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "express-body-parser-error-handler": "^1.0.4",
     "google-map-react": "^2.2.0",
     "jwt-decode": "^3.1.2",
-    "react-dotenv": "^0.1.3",
-    "express-body-parser-error-handler": "^1.0.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-hook-form": "^7.42.1",
```

Comment:
We don't need this "react-hook-form". Could you please delete it? `npm uninstall react-hook-form`